orion/sites/admin/src/programs/programEdit.js

```diff
@@ -178,9 +178,43 @@ export class ProgramEditPage extends React.Component {
 }
 
 function checkProgram(program) {
-  // *todo* change this later!
+  let checkpro=true;
+
+  let programId = program.programId
+
+  //checks if programId contains special characters
+  var iChars = "~`!#$%^&*+=-[]\\\';,/{}|\":<>?";
+  for (let i=0; i<programId.length; i++) {
```

Comment:
I think it's more clean to have spaces between = and <, >
`for (let i = 0; i < programId.length; i++) {`